.github/workflows/continuous-integration.yml

```diff
@@ -305,43 +302,34 @@ jobs:
     strategy:
       matrix:
         php-version:
-          - "7.4"
+          - "8.3"
         mariadb-version:
           # keep in sync with https://mariadb.org/about/#maintenance-policy
           - "10.0"  # Oldest version supported by DBAL
-          - "10.4"  # LTS (Jun 2024)
-          - "10.5"  # LTS (Jun 2025)
-          - "10.6"  # LTS (Jul 2026)
-          - "10.11" # LTS (Feb 2028)
-          - "11.1"  # STS (Aug 2024)
-          - "11.2"  # STS (Nov 2024)
-          - "11.3"  # STS (Feb 2025)
+          - "10.4"  # We have code specific to 10.4.3-10.5.2
+          - "10.5"  # We have code specific to 10.5.2-10.6.0
+          - "10.6"  # We have code specific to 10.6.0-10.10.0
+          - "10.11" # We have code specific to ^10.10
```

Comment:
I think it's better to combine the existing and the new comments instead of replacing the old one. They will be handy when it comes to removal of support of the versions no longer supported by the community.